foyer/opls_validation/chlorobenzene/chlorobenzene.gro

```diff
@@ -0,0 +1,2403 @@
+GROningen MAchine for Chemical Simulation
+ 2400
```

Comment:
Can we update this file to just have 1 cholorobenzene? Will make the testing a bit faster.